### DISCUSSION TITLE
Provide good default for the styles of the text editor

### DISCUSSION THREAD

**Discussion by gagarine** [zim-desktop-wiki/zim-desktop-wiki#2072] [Ideas]
This is how it looks by default. I believe this could be improve.
![Screenshot 2021-03-24 at 17 27 36](https://user-images.githubusercontent.com/164074/112346580-6290c480-8cc6-11eb-8c62-213b2da229e2.png)

In particular the readability is not very good.
- page top margin need to be bigger.
- max width (auto wrap)
- justify text
- space between line is not big enough
- space between paragraph is not big enough
- text is a bit to small
- The green of the title is out of place for me.

**Comment by gagarine** [comment]
Perhaps this is only linked to the macOS version? I'm using the macOS native bundle from #990

**Comment by neinseg** [comment]
No, AFAICT at least some of these points are generic to Windows and Linux too. An issue I found is that the way the GTK text editing component is integrated into Zim, it seems it cannot support separate configuration of paragraph spacing and spacing above/below headlines which would be a big upgrade, too.
